src/status_im/signals/core.cljs

```diff
@@ -79,13 +78,7 @@
              {:dispatch [:syncing/clear-states]}
 
              (and completed-pairing? receiver?)
-             {:dispatch [:profile.login/local-paired-user]}
-
-             (and error-on-pairing? (some? error))
-             {:dispatch [:toasts/upsert
-                         {:icon       :i/alert
-                          :icon-color colors/danger-50
-                          :text       error}]}))))
```

Comment:
Can we double check with design if we want to remove the toast?

Reply:
according to design flow and @VolodLytvynenko's comment it doesn't require any toast, [Figma](https://www.figma.com/file/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?type=design&node-id=3450-354380&mode=design&t=miSAbq1JaEDdWPPK-4)

Reply:
@ibrkhalil i revert this part, as QA team confirmed we should keep it